### ISSUE TITLE
Add Aromatic Ring descriptor using CDK

### ISSUE THREAD

**Issue by Kohulan** [Steinbeck-Lab/cheminformatics-python-microservice#84]
java code block from @JonasSchaub 
```
SmilesParser tmpSmiPar = new SmilesParser(DefaultChemObjectBuilder.getInstance());
IAtomContainer tmpMolecule = tmpSmiPar.parseSmiles("C1=CC=CC=C1");

//arguments might need to be changed to be in accordance with RDKit method
Aromaticity tmpAromaticity = new Aromaticity(ElectronDonation.daylight(), Cycles.cdkAromaticSet());

tmpAromaticity.apply(tmpMolecule);

//also, the cycle finder used here is debatable
IRingSet tmpMCBRings = Cycles.mcb(tmpMolecule).toRingSet();
int tmpNumberOfAromaticRings = 0;
for (IAtomContainer tmpRingContainer: tmpMCBRings.atomContainers()) {
    boolean tmpAreAllRingBondsAromatic = true;
    for (IBond tmpBond: tmpRingContainer.bonds()) {
        if (!tmpBond.isAromatic()) {
            tmpAreAllRingBondsAromatic = false;
            break;
        }
    }
    if (tmpAreAllRingBondsAromatic) {
        tmpNumberOfAromaticRings++;
    }
}
System.out.println(tmpNumberOfAromaticRings);
```

**Comment by JonasSchaub** [comment]
A few comments on that:

- The three parameters, 1) the electron donation model used for aromaticity detection, 2) the cycle finder used for aromaticity detection, and 3) the cycle finder used for the final ring detection (and count), most likely influence the output quite a lot. I have chosen the three options here initially which make sense generally and especially for comparability with RDKit (even though I haven't looked into their code), in my (humble) opinion. But I put them all up for debate. Best case scenario, they are going to be optional parameters that can be set.
- I have chosen to detect the aromatic rings via their aromatic bonds, not their aromatic atoms. Maybe also something debatable.
- Very minor: actually, SilentChemObjectBuilder is recommended over DefaultChemObjectBuilder. But I guess it doesn't matter here because this line is not going to be included in the actual code.

**Comment by JonasSchaub** [comment]
Required imports (without SmilesParser and SilentChemObjectBuilder):

import org.openscience.cdk.aromaticity.Aromaticity;
import org.openscience.cdk.aromaticity.ElectronDonation;

import org.openscience.cdk.graph.Cycles;

import org.openscience.cdk.interfaces.IAtomContainer;
import org.openscience.cdk.interfaces.IBond;
import org.openscience.cdk.interfaces.IRingSet;

**Comment by Kohulan** [comment]
fixed with #90
